frontend/packages/dev-console/src/utils/application-utils.ts

```diff
@@ -44,6 +48,29 @@ export const edgesFromAnnotations = (annotations): string[] => {
   return edges;
 };
 
+export const edgesFromServiceBinding = (
+  source: K8sResourceKind,
+  sbrs: K8sResourceKind[],
+): string[] => {
+  const edges: string[] = [];
+  _.forEach(sbrs, (sbr) => {
+    let edgeExists = false;
+    if (_.get(sbr, 'spec.applicationSelector.resourceRef') === source.metadata.name) {
```

Comment:
You also need to match the other properties like `resource` to ensure it's the right kind.